core/src/main/scala/filodb.core/memstore/TimeSeriesPartition.scala

```diff
@@ -57,7 +57,7 @@ final case class InfoAppenders(info: ChunkSetInfo, appenders: TimeSeriesPartitio
  */
 class TimeSeriesPartition(val partID: Int,
                           val schema: Schema,
-                          partitionKey: BinaryRegion.NativePointer,
+                          override val partitionKey: BinaryRegion.NativePointer,
```

Comment:
I still don't understand this part.  `partitionKey` was not a part of ReadablePartiton previously, why is it needed now?  It is duplicate, because that's exactly what `partKeyOffset` is.   We should not add an additional member to TSPartition, when it is redundant.  It adds extra memory usage for each TSPartition object.